src/pages/documentation/adobe-journey-optimizer-decisioning/release-notes.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## February 2, 2023
+
+### Android Optimize 2.0.0
+
+* Initial release to support [Adobe Journey Optimizer - Decisioning](./index.md) for Adobe Experience Platform Mobile SDKs for Android compatible with Mobile Core 2.0.0. This library is available as an [open sourced project on GitHub](https://github.com/adobe/aepsdk-optimize-android).
```

Comment:
I have a separate pr (https://github.com/AdobeDocs/aep-mobile-sdkdocs/pull/234) for Optimize release notes as it is not an initial release for this extension.